src/contradictory_claims/data/preprocess_cord.py

```diff
@@ -12,7 +19,7 @@ def filter_metadata_for_covid19(metadata_path: str, virus_lex_path: str, pub_dat
     :param metadata_path: path to CORD-19 metadata.csv file
     :param virus_lex_path: path to COVID-19 lexicon
     :param pub_date_cutoff: cut-off for publication date in the format 'yyyy-mm-dd'
```

Comment:
Might want to change in the future to requiring a Datetime in the input, maybe that would be more robust?

Reply:
Yeah that's a good point. I can add a type check in the function to throw an error if it's not datetime type.